package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Machine-readable custom errors for Apollostack's GraphQL server",
   "main": "dist/index.js",
   "scripts": {
-    "test": "make test"
+    "test": "make test",
+    "build": "tsc"
```

Comment:
Let's stick to the makefile for the build step.  I added the test script to run `make test` just to keep things canonical with `npm init`